Source/VCGeneration/ManualSplitFinder.cs

```diff
@@ -1,18 +1,21 @@
+#nullable enable
 using System.Collections.Generic;
 using System.Diagnostics.Contracts;
 using System.Linq;
+using System.Threading.Tasks.Dataflow;
```

Comment:
This seems like a very relevant library for this kind of task, but I don't actually see how it's used. Is it?